src/absil/ilsupp.fs

```diff
@@ -676,10 +676,10 @@ let linkNativeResources (unlinkedResources:byte[] list)  (ulLinkedResourceBaseRV
                 // Conversion was successful, so read the object file
                 objBytes <- FileSystem.ReadAllBytesShim(tempObjFileName) ; 
                 //Array.Copy(objBytes, pbUnlinkedResource, pbUnlinkedResource.Length)
-                System.IO.File.Delete(tempObjFileName)
+                FileSystem.FileDelete(tempObjFileName)
```

Comment:
Several of the changes are just adjusting the Visual F# Tools code to use the file system shim